src/en/ohjoysextoy/src/eu/kanade/tachiyomi/extension/en/ohjoysextoy/OhJoySexToy.kt

```diff
@@ -0,0 +1,135 @@
+        title = element.selectFirst(".comicthumbdate")!!.text().substringBefore(" by")
+        thumbnail_url = element.selectFirst(".comicarchiveframe > a > img")?.absUrl("src")
+    }
+
+    override fun latestUpdatesNextPageSelector(): String? = null
+
+    // Search
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request = GET("$baseUrl/?s=$query", headers)
+
+    override fun searchMangaSelector(): String = "h2.post-title"
+
+    override fun searchMangaFromElement(element: Element): SManga = SManga.create().apply {
+        setUrlWithoutDomain(element.selectFirst("a")!!.absUrl("href"))
+        title = element.selectFirst("a")!!.text().substringBefore(" by")
+    }
+
+    override fun searchMangaNextPageSelector(): String? = null
+
+    // etc
+
+    override fun mangaDetailsParse(document: Document): SManga = SManga.create().apply {
+        thumbnail_url = document.selectFirst("meta[property=\"og:image\"]")
+            ?.absUrl("content")
+        status = SManga.COMPLETED
+        title = document.selectFirst("meta[property=\"og:title\"]")
+            ?.attr("content")
+            ?.substringBefore(" by")
+            ?: ""
+        author = document.selectFirst("meta[property=\"og:title\"]")
+            ?.attr("content")
+            ?.substringAfter("by ", "")
+            ?: ""
+        description = parseDescription(document)
+        genre = document.select("meta[property=\"article:section\"]:not(:first-of-type)")
+            .eachAttr("content")
+            .joinToString()
+        update_strategy = UpdateStrategy.ONLY_FETCH_ONCE
+        setUrlWithoutDomain(
+            document.selectFirst("meta[property=\"og:url\"]")
+                ?.absUrl("content")
+                ?: "",
+        )
+    }
+
+    private fun parseDescription(document: Document): String {
+        val desc = document.selectFirst("meta[property=\"og:description\"]")
+            ?.attr("content")
+            ?.split(MULTI_SPACE_REGEX)
+            ?.get(0) + "..."
+
+        val authorCredits = document.select(".entry div.ui-tabs div a")
+            .joinToString("\n") { link ->
+                "${link.text()}: ${link.absUrl("href")}"
+            }
+
+        return listOf(desc, authorCredits, "(Full description and credits in WebView)").joinToString("\n\n")
+    }
+
+    override fun chapterListRequest(manga: SManga): Request = GET("$baseUrl${manga.url}", headers)
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val document = response.asJsoup()
+        val dateString = document.selectFirst(".post-date")?.text() ?: ""
+
+        return listOf(
+            SChapter.create().apply {
+                name = document.title()
+                scanlator = document.selectFirst(".post-author a")?.text()
+                date_upload = SimpleDateFormat("MM/dd/yyyy", Locale.ENGLISH).tryParse(dateString)
```

Comment:
save the `SimpleDateFormat` object in class to avoid initializing it on every chapter